packages/omgidl-serialization/src/MessageReader.test.ts

```diff
@@ -1929,7 +1929,7 @@ module builtin_interfaces {
     };
     const writer = new CdrWriter({ kind: EncapsulationKind.PL_CDR2_LE });
     writer.dHeader(4 + 4 + 4 + 4 + 4 + 1);
-    writer.emHeader(false, 0, 4);
+    writer.emHeader(false, 101, 4);
```

Comment:
what made this change necessary?

Reply:
the schema is:
```
      @mutable
      struct Message {
        @id(100) uint8 bittybyte;
        float floaty;
        @id(300) uint32 bytier;
      };
```
so with the new id calculation, `floaty` gets an id of 101 instead of 0

Reply:
floaty is pleased